src/main.ts

```diff
@@ -208,6 +211,30 @@ async function run(): Promise<void> {
       await octokit.graphql<AddLabelsMutation>({ query: print(AddLabels), ...addLabelsMutationData });
     }
 
+    const reviewers = (core.getInput('reviewers') || '')
+      .split(',')
+      .map((reviewer) => reviewer.trim())
+      .filter((reviewer) => !!reviewer);
+    const teamReviewers = (core.getInput('teams') || '')
+      .split(',')
+      .map((team) => team.trim())
+      .filter((team) => !!team);
+    if (reviewers.length > 0 || teamReviewers.length > 0) {
+      core.info('adding reviewers to the PR');
+      const addReviewersMutationData: AddReviewersMutationVariables = {
+        input: {
+          pullRequestId: browserslistUpdatePR,
```

Comment:
You should not reference `browserslistUpdatePR` there, but `prId` instead. `browserslistUpdatePR` is defined only if there was already an existing PR which then gets updated.

Reply:
You are right sorry :/ I didn't see the error from the compiler the first time. But I just saw it when opening my code editor now. Anyway, it's fixed.